### DISCUSSION TITLE
Hintak’s fork of `Font-Validator` has been archived

### DISCUSSION THREAD

**Discussion by jlarmstrongiv** [fonttools/fontbakery#4327] [General]
Will font bakery maintain their own fork?

**Comment by felipesanches** [comment]
No. There's no interest in maintaining a fork of that. FontBakery's approach, [as described](https://font-bakery.readthedocs.io/en/latest/product/Overview.html#what-is-the-opportunity-for-designers-and-foundries-to-collaborate-on-font-quality-assurance) in its documentation, is to include detailed rationale descriptions for each check to justify why they're needed. Font Validator does not do that, so it should ideally be [entirely substituted](https://github.com/fonttools/fontbakery/issues/1541) by new fontbakery checks. Please open issues suggesting porting of any check from Font Validator that may be not yet available in Font Bakery.

For some time we may still keep the [`fontvalidator` profile](https://github.com/fonttools/fontbakery/blob/2e58c06a203d285c55be82a083a9cd393aca7207/Lib/fontbakery/profiles/fontval.py#L25), which simply wraps around the fontval binary, if available. But I suspect not a lot of people actually uses it. (I may be wrong, so please let me know if any of you do use it!)

**Reply by davelab6** [reply]
We could also do the large task of porting all (in a prioritized way) the fontval checks to python as native fontbakery checks :)
